tests/Functional/Command/MauticLanguagePackerCommandTest.php

```diff
@@ -0,0 +1,608 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Functional\Command;
+
+use App\Command\MauticLanguagePackerCommand;
+use App\Service\BuildPackageService;
+use App\Service\Transifex\ResourcesService;
+use App\Service\UploadPackageService;
+use App\Tests\Common\Client\MockResponse;
+use App\Tests\Common\Trait\ResponseBodyBuilderTrait;
+use Aws\MockHandler as AwsMockHandler;
+use Aws\Result;
+use Aws\ResultInterface;
+use GuzzleHttp\Handler\MockHandler;
+use PHPUnit\Framework\Assert;
+use Psr\Http\Message\ResponseInterface;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\HttpFoundation\Request;
+
+class MauticLanguagePackerCommandTest extends KernelTestCase
+{
+    use ResponseBodyBuilderTrait;
+
+    private CommandTester $commandTester;
+
+    private Filesystem $filesystem;
+
+    private string $packagesDir;
+
+    private string $translationsDir;
+
+    private MockHandler $mockHandler;
+
+    private AwsMockHandler $awsMockHandler;
+
+    protected function setUp(): void
+    {
+        $container        = self::getContainer();
+        $parameterBag     = $container->get('parameter_bag');
+        $this->filesystem = $container->get('filesystem');
+
+        $application = new Application();
+        $application->add(
+            new MauticLanguagePackerCommand(
+                $parameterBag,
+                $this->filesystem,
+                $container->get(ResourcesService::class),
+                $container->get(BuildPackageService::class),
+                $container->get(UploadPackageService::class)
+            )
+        );
+        $command             = $application->find(MauticLanguagePackerCommand::NAME);
+        $this->commandTester = new CommandTester($command);
+
+        $this->packagesDir     = $parameterBag->get('mlp.packages.dir');
+        $this->translationsDir = $parameterBag->get('mlp.translations.dir');
+
+        $this->mockHandler    = $container->get('http.client.mock_handler');
+        $this->awsMockHandler = $container->get('aws.client.mock_handler');
```

Comment:
```suggestion
        $this->mockHandler    = $container->get(MockHandler::class);
        $this->awsMockHandler = $container->get(AwsMockHandler::class);
```
This is connected with the suggestions in `services_test.yaml`.